transport/ice/IceSession.cpp

```diff
@@ -1613,20 +1965,22 @@ void IceSession::CandidatePair::processTimeout(const uint64_t now)
     if (state == ProbeState::Succeeded)
     {
         if (nominated &&
-            (utils::Time::diffGE(receptionTimestamp, now, _config.hostProbeTimeout * utils::Time::ms) ||
+            (utils::Time::diffGE(receptionTimestamp, now, _iceSession->_config.hostProbeTimeout * utils::Time::ms) ||
                 pendingTransactionCount() > 2))
         {
-            logger::debug("restart prob due to lack of inbound traffic", _name.c_str());
-            restartProbe(now);
+            logger::info("restart prob due to lack of inbound traffic", _iceSession->_logId.c_str());
+            restartProbe(now, false);
```

Comment:
When we see lack of traffic on the nominated, we will restart probe without remove the nomination. only after a few time, we consider unreachable and then we will remove the nomination and change the state to `CONNECTING`